maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -424,8 +424,27 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         waitForAppToSettle()
     }
 
-    fun openLink(link: String) {
-        driver.openLink(link)
+    fun openLink(link: String, autoVerify: Boolean) {
+        driver.openLink(link, autoVerify)
+        waitForAppToSettle()
+    }
+
+    fun autoVerifyApp(appId: String?, autoVerifyApp: (text: String) -> Unit) {
+        if (driver is AndroidDriver) {
+            driver.autoVerifyApp(appId, autoVerifyApp)
+            waitForAppToSettle()
+        }
+    }
+
+    fun autoVerifyChromeAgreement(verifyGoogleChromeAgreement: () -> Unit) {
```

Comment:
This top-level method is:

- Leaking OS-specific details (it is Android-only)
- Is always called in tandem with `autoVerifyApp`

I think it can simply be a part of the `autoVerifyApp` method

Reply:
Done added this in AndroidDriver. This is no more leaking in Maestro or Orchestra class 💯